docs/source/java/install.rst

```diff
@@ -0,0 +1,148 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+Installing Java Modules
+=======================
+
+System Compatibility
+--------------------
+
+Java modules are regularly built and tested on macOS and Linux distributions.
+
+Java Compatibility
+------------------
+
+Java modules is currently compatible with JDK 8 / 9 / 10 / 11.
+
+Using Maven
+-----------
+
+Downloading in Maven is triggered by a project declaring a dependency that is not present in the local repository.
+
+Central Repository
+******************
+
+By default, Maven will download from the central repository: https://repo.maven.apache.org/maven2/org/apache/arrow/
+
+Configure your pom.xml with java module version needed. For example:
+
+.. code-block::
+
+    <dependency>
+        <groupId>org.apache.arrow</groupId>
+        <artifactId>arrow-vector</artifactId>
+        <version>7.0.0</version>
+    </dependency
```

Comment:
There's a missing bracket here, also, shouldn't we show the 'full' set of dependencies needed? i.e. memory-netty, format, and vector? We also don't need to explain Maven or pom.xml here. Just state what modules are needed and show the example XML.

Reply:
Added libraries and xml demo